workflow/scripts/create_somalier_mqc_config.py

```
@@ -400,80 +486,96 @@ def get_expected_relatedness(sample_a, sample_b, ped_df):
-        else:
-            somalier_rel_config = {}
-
-        write_somalier_mqc(
-            rel_check_df,
-            somalier_rel_config,
-            args.rel_check_mqc
-        )
+            # Create sex check tsv
+            try:
+                sex_check_df = get_sex_check_df(args.samples)
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)